sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareEntityRelationType.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// HealthcareEntityRelationType
+    /// </summary>
+    public class HealthcareEntityRelationType : IEquatable<HealthcareEntityRelationType>
+    {
+        /// <summary>
+        /// Specifies that the entity corresponds to a Person.
+        /// </summary>
+        public static readonly HealthcareEntityRelationType DirectionOfBodyStructure = new HealthcareEntityRelationType("DirectionOfBodyStructure");
```

Comment:
Just curious why you are specifying exact relation types here.  I originally thought we could create an enum for this in the Swagger but the service owner responded to my comment by saying that even he didn't have an exhaustive list of possible relation types ([source](https://github.com/Azure/azure-rest-api-specs/pull/12338#discussion_r563926343)).

Reply:
@abhahn  Should i change this to just string ?

Reply:
I'm not really opposed to using an enum if we can ensure that we have a complete set of values.  But if an exhaustive list is too large or something we aren't able to find from the science team that developed the model then maybe a string would be better.

Reply:
The list doesn't have to be complete as extensible enums can keep growing. I will push on the science team to have at least the common relationship types that they are communicating to the user in other forms (like docs.microsoft) and add them to the swagger so the SDKs can use that.
(adding values to the extensible enum is a non-breaking change)

Reply:
Looks like they are going to add it to the swagger. yay! u can leave as is for the amount of relationships if u want.